test/e2e/framework/framework.go

```diff
@@ -247,9 +247,18 @@ func (f *Framework) BeforeEach() {
 
 	if TestContext.GatherKubeSystemResourceUsageData != "false" && TestContext.GatherKubeSystemResourceUsageData != "none" {
 		var err error
+		var nodeMode NodesSet
+		switch TestContext.GatherKubeSystemResourceUsageData {
+		case "master":
+			nodeMode = MasterAndDNSNodes
```

Comment:
Shouldn't this be just master?

Reply:
That’s just a tenporary change to test the collection inside this PR only. If it works, Next we will add coredns as default in kubeup in this PR (and als bump memory limit), and rerun the scale test again to observe coreDNS memory usage.